app/src/main/java/de/westnordost/streetcomplete/quests/QuestUtil.kt

```diff
@@ -3,23 +3,27 @@ package de.westnordost.streetcomplete.quests
 import android.content.res.Resources
 import android.text.Html
 import android.text.Spanned
-
+import androidx.core.os.ConfigurationCompat
+import androidx.core.text.HtmlCompat
 import de.westnordost.osmapi.map.data.Element
 import de.westnordost.osmfeatures.FeatureDictionary
-import de.westnordost.streetcomplete.data.quest.QuestType
 import de.westnordost.streetcomplete.data.osm.osmquest.OsmElementQuestType
+import de.westnordost.streetcomplete.data.quest.QuestType
 import java.util.*
 import java.util.concurrent.FutureTask
 
 fun Resources.getQuestTitle(questType: QuestType<*>, element: Element?, featureDictionaryFuture: FutureTask<FeatureDictionary>?): String {
-    val arguments = getTemplateArguments(questType, element, configuration.locale, featureDictionaryFuture)
+    val locale = ConfigurationCompat.getLocales(configuration)[0]
+    val arguments = getTemplateArguments(questType, element, locale, featureDictionaryFuture)
```

Comment:
Well actually, the deprecation of `configuration.locale` is there because one is supposed to take into account that the user may have a chain of locales to apply - if a resource is not found for the first locale, then the second locale should be used, and so forth.

So this change only silences the warning but doesn't solve the problem, `getTemplateArguments` would need to accept a list of locales and go through all the given locales until one name has been found.

Reply:
In that case, `getTemplateArguments` could be changed to accept `LocaleListCompat` instead.

Reply:
Yes, and do some for loop or something within the lazy until we reached the end of the LocaleList or a name was found

Reply:
(maybe outsource that logic from the lazy into an own function, might get a little hard to read otherwise)